config/routes.rb

```diff
@@ -11,9 +11,11 @@
   root to: redirect('/candidate')
 
   get '/accessibility', to: 'content#accessibility'
+  get '/candidate/terms-of-use', to: 'content#terms_candidate', as: :terms_candidate
```

Comment:
We'll need to update the link in the signup screen as well.

Reply:
Have updated 👍🏻